openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -228,8 +228,7 @@ def setUpClass(cls):
                 graded=False,
             )
             cls.graded_score_weight_blocks[(graded, has_score, weight)] = block
-
-        host = os.environ.get('BOK_CHOY_HOSTNAME', '127.0.0.1')
+        host = os.environ.get('127.0.0.1')
```

Comment:
```suggestion
        host = '127.0.0.1'
```

You don't need the `os.environ.get` if you aren't fetching something from an environment variable.